PxtlCa.XmlCommentMarkDownGenerator/TagRenderers.cs

```diff
@@ -29,6 +29,14 @@ public TagRenderer(string formatString, Func<XElement, ConversionContext, IEnume
                         x.Element("members").Elements("member").ToMarkDown(context.MutateAssemblyName(x.Element("assembly").Element("name").Value))
                 }
             ),
+            ["br"] = new TagRenderer(
+                "\n",
```

Comment:
For proper markdown, shouldn't it be "\n  " with two trailing space afterwards?  Then it's compatible with standard-markdown in addition to github-flavored markdown.

https://stackoverflow.blog/2009/10/15/markdown-one-year-later/